server/src/test/scala/za/co/absa/atum/server/api/repository/PartitioningRepositorySpec.scala

```diff
@@ -62,18 +62,18 @@ class PartitioningRepositorySpec extends ZIOSpecDefault with TestData {
   // Get Partitioning Measures Mocks
   private val getPartitioningMeasuresMock = mock(classOf[GetPartitioningMeasures])
 
-  when(getPartitioningMeasuresMock.apply(partitioningSubmitDTO1)).thenReturn(ZIO.succeed(Seq()))
-  when(getPartitioningMeasuresMock.apply(partitioningSubmitDTO2)).thenReturn(ZIO.fail(new Exception("boom!")))
-  when(getPartitioningMeasuresMock.apply(partitioningSubmitDTO3)).thenReturn(ZIO.fail(new Exception("boom!")))
+  when(getPartitioningMeasuresMock.apply(partitioningDTO1)).thenReturn(ZIO.succeed(Seq()))
+  when(getPartitioningMeasuresMock.apply(partitioningDTO2)).thenReturn(ZIO.fail(new Exception("boom!")))
+  when(getPartitioningMeasuresMock.apply(partitioningDTO3)).thenReturn(ZIO.fail(new Exception("boom!")))
 
   private val getPartitioningMeasuresMockLayer = ZLayer.succeed(getPartitioningMeasuresMock)
 
   // Get Partitioning Additional Data Mocks
   private val getPartitioningAdditionalDataMock = mock(classOf[GetPartitioningAdditionalData])
 
-  when(getPartitioningAdditionalDataMock.apply(partitioningSubmitDTO1)).thenReturn(ZIO.succeed(Seq()))
-  when(getPartitioningAdditionalDataMock.apply(partitioningSubmitDTO2)).thenReturn(ZIO.fail(new Exception("boom!")))
-  when(getPartitioningAdditionalDataMock.apply(partitioningSubmitDTO3)).thenReturn(ZIO.fail(new Exception("boom!")))
+  when(getPartitioningAdditionalDataMock.apply(partitioningDTO1)).thenReturn(ZIO.succeed(Seq()))
+  when(getPartitioningAdditionalDataMock.apply(partitioningDTO2)).thenReturn(ZIO.fail(new Exception("boom!")))
```

Comment:
Same as above, use it to check on return of multiple items in the sequence.

Reply:
Done